apps/website-new/docs/en/guide/basic/runtime.mdx

```diff
@@ -449,6 +449,31 @@ registerPlugins([runtimePlugin()]);
 
 If you want to develop Module Federation plugin, you can read [Module Federation Plugin System](../../plugin/dev/index) for more info.
 
+## FederationHost
+
+In addition to exposing APIs, Federation Runtime also provides the FederationHost class, which you can use to create a FederationHost instance.
+
+* When to use `FederationHost` ?
```

Comment:
it would be a good idea to have small section on "What is FederationHost" 

How i explained it to people before was that in the __FEDERATION__.__INSTANCES__ - each runtime or application who uses module federation creates a FederationHost instance, it represents an both consumer and provider instances of the system. Or something so users understand what federation host actually is for